### ISSUE TITLE
Add Window menu that lists open sketch windows [imported]

### ISSUE THREAD

**Issue by cmaglie** [arduino/Arduino#234]
This is [Issue 234](http://code.google.com/p/arduino/issues/detail?id=234) moved from a Google Code project.
Added by 2010-04-26T13:54:46.000Z by [follower@gmail.com](http://code.google.com/u/follower@gmail.com/).
Please review that bug for more context and additional comments, but update this bug.

Original labels: Type-Enhancement, Priority-Medium
### Original description

**What change would like to see?**

I would like a "Window" menu added to the menu bar.

**Why?**

Now that we can have multiple sketches open at one time this would make it easier to swap 
between them instead of cycling through the windows one-by-one. 

Would this cause any incompatibilities with previous versions?

No.

But this is mostly an issue when you have many sketches open which may be considered a non-
typical user scenario. However the "Window" menu is relatively common in desktop applications.

If so, how can these be mitigated?

N/A

**Comment by lmihalkovic** [comment]
simple first step. anything else reeeeally important to add?

<img width="418" alt="screen shot 2015-11-26 at 10 35 43 am" src="https://cloud.githubusercontent.com/assets/1318077/11419884/a407e3ac-942a-11e5-98a4-12ff8fe3595f.png">

**Comment by lmihalkovic** [comment]
better without the app name in the title?

<img width="412" alt="screen shot 2015-11-26 at 10 49 07 am" src="https://cloud.githubusercontent.com/assets/1318077/11420005/706887b2-942b-11e5-9b49-0e2914c3ccf4.png">

**Comment by lmihalkovic** [comment]
it seems many apps on Mac at least have these.. dunno about windows

<img width="302" alt="screen shot 2015-11-26 at 11 15 23 am" src="https://cloud.githubusercontent.com/assets/1318077/11420549/1e7b9576-942f-11e5-8f8b-d247f21ab26c.png">

**Comment by lmihalkovic** [comment]
@mralexgray  *2010* it seem...  Window menus are largely overrated anyhow... ;-)

**Comment by thomasaw** [comment]
This is a standard Mac app feature. I hope it gets merged in.

**Comment by Breidenbach** [comment]
I agree with thomasaw - having used Macs forever (and the Lisa before that) I miss this feature.

**Comment by ffissore** [comment]
@thomasaw @Breidenbach unfortunately there is nothing to merge. This is just an issue. I look forward for a PR that I'll be happy to test and merge

**Comment by lmihalkovic** [comment]
One of the miracles of open source is to be able to shift responsibilities around by *pushing* with a large smile and a friendly *PR-welcome*, piles of loosely taped together code that users are expected to finish themselves.

**Comment by nesdnuma** [comment]
Any chance to see this issue solved on a forthcoming update?

**Comment by erathos** [comment]
May be for christmas ? Waiting for that patch eargerly since 5 years. Unable to do it myself. Thanks to our future savior

**Comment by lmihalkovic** [comment]
downloaded the latest UI today. looking at the 1.9.0 code, it seems it still does not address this ... 2012 .. what's the bounty on this one?! ;-)

<img width="920" alt="screen shot 2018-04-28 at 9 38 43 pm" src="https://user-images.githubusercontent.com/1318077/39400228-c4f91bba-4b2c-11e8-92f9-cbac23651ddc.png">

**Comment by kosso** [comment]
Amazed that this issue is six years old. 
This is fairly standard macOS / OSX user interface for multiple windows. 

(The Windows version also lacking it.)

Any way to bump this?
